recipes/oras-py/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "oras-py" %}
```

Comment:
Is this the same as https://pypi.org/project/oras? If so can we use the PyPI name to avoid user confusion?

Reply:
No it's not - oras is the go client, and this is the python client. The pypi name is also oras. It's similar to docker - you would "pip install docker" https://pypi.org/project/docker/ but on here it's https://anaconda.org/conda-forge/docker-py. If a rust library is ever made you could also eventually get oras-rs.

Reply:
Oh sorry - so that is the correct project on Pypi, but I  mean to say that the core "oras" has a go implementation that builds to just "oras."

Reply:
Here is the repo for that: https://github.com/oras-project/oras

Reply:
So to be more clear:
- https://github.com/oras-project/oras is the main client, compiles to "oras" (in Go) and any oras on here should be for that.
- https://github.com/oras-project/oras-py is the Python version, and is on pypi as "oras" and added as oras-py here
- https://github.com/oras-project/oras-rs is a potential rust version, not sure about it's status, but could be a thing in the future if it's picked up to work on.

Specifically I was chatting with @wolfv and because the python variant is important for some cool project ideas I think it would be a good idea to add here, but we do have to respect the namespace of the project! I can ping the other oras developers and see if we can add the traditional oras as just "oras."

Reply:
If I understood it correctly the package in this PR is `oras-py` then, right? If so it should be renamed to avoid confusion and avoid taking the namespace of the go package. Also, can you use the PyPI sdist for it instead of GH?

Reply:
I don't think I follow - this package is oras-py, and it's oras on pypi because there can only be one oras there (in Python) and we don't need to save a space for the go client,  but that's not the case here - we  need to reserve "oras" for the go client and this should be oras-py. It's exactly the same convention for the docker pypi client (docker-py on here).

Reply:
Yeah, I think it's fine to call this one `oras-py` on here. I've actually released the `oras` (Go) client on conda-forge a couple weeks ago, so that name is taken anyways :)